python/pyarrow/compute.py

```diff
@@ -656,8 +673,13 @@ def top_k_unstable(values, k, sort_keys=None, memory_pool=None):
     Parameters
     ----------
     values : Array, ChunkedArray, RecordBatch, or Table
-    k :  The number of `k` elements to keep.
-    sort_keys : Column key names to order by when input is table-like data.
+        Data to sort and get top indices from.
+    k : int
+        The number of `k` elements to keep.
+    sort_keys : List or tuple
```

Comment:
```suggestion
    sort_keys : list
```

Or "list-like" in you prefer. Since it supports any sequence, I don't think it's needed to specifically mention "tuple" in addition to list.

(reason this catched my attention is actually because I first misread it as "list of tuple", while `sort_keys` in general follow that structure, while here it should be just be a list of strings)